modules/oe_whitelabel_search/src/Form/SearchForm.php

```diff
@@ -48,40 +48,62 @@ public function buildForm(array $form, FormStateInterface $form_state, array $co
     $form_state->set('oe_whitelabel_search_config', $config);
     $input_value = '';
 
+    $form['#attributes']['class'][] = 'bcl_search_form';
+
     if (!empty($config['input']['name'])) {
       $input_value = $this->getRequest()->get($config['input']['name']);
     }
 
     $form['#action'] = $config['form']['action'];
 
     $form['search_input'] = [
+      '#prefix' => '<div class="bcl-search-form__group">',
+      '#suffix' => '</div>',
       '#type' => 'textfield',
       '#title' => $config['input']['label'],
       '#title_display' => 'invisible',
+      '#size' => 20,
       '#attributes' => [
         'placeholder' => $config['input']['placeholder'],
         'class' => [
           $config['input']['classes'],
+          'border-start-0',
+          'rounded-0',
+          'rounded-start',
         ],
       ],
       '#default_value' => $input_value,
       '#required' => TRUE,
     ];
 
-    $form['submit'] = [
-      '#prefix' => '<div class="ms-2">',
-      '#suffix' => '</div>',
-      '#type' => 'submit',
-      '#name' => FALSE,
-      '#value' => $config['button']['label'],
-      '#attributes' => [
-        'class' => [
-          'btn-md',
-          $config['button']['classes'],
+    $button = [
+      '#type' => 'pattern',
+      '#id' => 'button',
+      '#variant' => 'light',
+      '#fields' => [
+        'settings' => [
+          'type' => 'submit',
+        ],
+        'icon' => [
+          'name' => 'search',
+          'size' => 'xs',
```

Comment:
Add this here to check if attributes are working:
'attributes' => new Attribute('class', ['align-self-center']),

At the beginning of file add:
use Drupal\Core\Template\Attribute;